system/hardware/tici/hardware.py

```diff
@@ -440,6 +440,7 @@ def initialize_hardware(self):
     affine_irq(1, 7)    # msm_drm
     affine_irq(1, 250)  # msm_vidc
     affine_irq(1, 8)    # i2c_geni (sensord)
+    affine_irq(1, 336)    # lsm gpio (sensord)
```

Comment:
need to do this after setting up the gpio interface since the interrupt isn't setup until then

Reply:
done